lib/connection.ts

```diff
@@ -253,7 +253,7 @@ export class Connection extends Entity {
     }
 
     this.options = this._connection.options;
-    this.options.operationTimeoutInSeconds = options?.operationTimeoutInSeconds ?? defaultOperationTimeoutInSeconds;
+    this.options.operationTimeoutInSeconds = this.options?.operationTimeoutInSeconds ?? defaultOperationTimeoutInSeconds;
```

Comment:
I am not sure which one we should take when both `this._connection.options.operationTimeoutInSeconds` and `options.operationTimeoutInSeconds` are valid.